core/modules/user/css/user-simplified-page.css

```diff
@@ -0,0 +1,52 @@
+/**
+ * Simplified login pages.
+ */
+
+.user-simplified-page-logo {
+  text-align: center;
+}
+.user-simplified-page-logo .logo {
+  display: inline-block;
+  max-width: 240px;
```

Comment:
Great to have a max-width for the logo. In my opinion, it could be a bit smaller, e.g. 200px or 160px. If this complicates things just before merging, we can certainly discuss it in a follow-up – then maybe also providing a max-width for the maintenance page.

Reply:
I left this at the current width because a lot of logos tend to be horizontal, i.e.
![image](https://github.com/user-attachments/assets/3bc26519-562a-43a9-a795-c8442c236724)
![image](https://github.com/user-attachments/assets/59d7595a-9e12-4bd0-8bbd-55407917cb43)

Even with a square logo, a 240px width (and thus 240px height) makes a form that fits easily on a mobile phone screen with no scrolling.

Reply:
I agree, for horizontal logos 240px is perfect. Square logos look quite big with this size, even if they fit on a phone screen. An approach to cover both use cases could be to set max-width of 240px to the link and a max-height of e.g. to the image. (Still fine if we won't do it last minute in this PR.)

Reply:
We can create that as a follow-up too.